### DISCUSSION TITLE
Welcome - Share your feedback!

### DISCUSSION THREAD

**Discussion by rien** [dodona-edu/dolos#713] [Announcements]
Welcome to the discussion page of the source code plagiarism detection tool **Dolos**. 
Feel free to share your experience with Dolos and give us some feedback.

If you are having difficulties running an analysis or have a question about something else, feel free to create a new topic. Our documentation website can be found at https://dolos.ugent.be.

If you encounter a bug, you can create a new issue directly, or discuss the problem here first. Feature requests are welcome as well.

**Comment by ramrod1460** [comment]
Has there been any discussion regarding support for the **_Groovy language_**. Perhaps Groovy support is covered under Java ...just not sure. I would like to use DOLOS ( or a similar tool ) to ensure that code submissions I receive are not plagiarized in whole or in part.

**Reply by rien** [reply]
Hi @ramrod1460, we can definitely add support for Groovy.  This is not much work, so you can expect support to land in our next release. I've created an issue so you can track this: https://github.com/dodona-edu/dolos/issues/1504

**Reply by ramrod1460** [reply]
Thanks so much ...wonderful news !

**Reply by ramrod1460** [reply]
Would this forum be the appropriate place to monitor for the release ( _to support Groovy_ ) or is there a way I can sign up to receive an eMail or other type of notification when the Groovy support is implemented ?
